guides/source/contributing_to_ruby_on_rails.md

```diff
@@ -715,47 +726,41 @@ that you don't already have.
 
 ### Older Versions of Ruby on Rails
 
-If you want to add a fix to older versions of Ruby on Rails, you'll need to set up and switch to your own local tracking branch. Here is an example to switch to the 4-0-stable branch:
+If you want to add a fix to versions of Ruby on Rails older than the next release, you'll need to set up and switch to your own local tracking branch. Here is an example to switch to the 7-0-stable branch:
 
 ```bash
-$ git branch --track 4-0-stable rails/4-0-stable
-$ git checkout 4-0-stable
+$ git branch --track 7-0-stable rails/7-0-stable
+$ git checkout 7-0-stable
 ```
 
-TIP: You may want to [put your Git branch name in your shell prompt](http://qugstart.com/blog/git-and-svn/add-colored-git-branch-name-to-your-shell-prompt/) to make it easier to remember which version of the code you're working with.
-
-NOTE: Before working on older versions, please check the [maintenance policy](maintenance_policy.html).
+NOTE: Before working on older versions, please check the [maintenance policy](maintenance_policy.html). No changes will not be accpeted to versions that have reached end of life.
```

Comment:
```suggestion
NOTE: Before working on older versions, please check the [maintenance policy](maintenance_policy.html). No changes will be accepted to versions that have reached end of life.
```

Reply:
"Changes will not be accepted to versions that have reached end of life." reads a bit easier